tensorflow/compiler/xla/service/gpu/nvptx_compiler.cc

```diff
@@ -234,6 +237,48 @@ bool MaybeLoadPtxFromFile(const HloModuleConfig module_config,
   return false;
 }
 
+std::unique_ptr<llvm::Module> MaybeLoadLLVMFromFile(const HloModule* module, llvm::Module* llvm_module) {
+  // If the xla_gpu_llvm_ir_file option is set, be explicit if a file is used
+  // and warn when a file is not used to ease catching typo in filename.
+  if (module == nullptr) {
+    return nullptr;
+  }
+
+  std::string prefix = xla::FilenameFor(*module, "", "");
+  std::string matched_filename;
+  for (const string& full_filename :
+       module->config().debug_options().xla_gpu_llvm_ir_file()) {
+    // To ease comparing many LLVM versions, accept different suffixes then
+    // the original filename.
+    auto filename = tensorflow::io::Basename(full_filename);
+    if (absl::StartsWith(filename, prefix)) {
+      matched_filename = full_filename;
+      VLOG(0) << "RunBackend() - Will load LLVM from file: " << full_filename;
+      break;
+    }
+  }
+  if (module->config().debug_options().xla_gpu_llvm_ir_file().size() > 0 &&
+      matched_filename.empty()) {
+    VLOG(0) << "RunBackend() - For module with prefix '" << prefix
```

Comment:
I'm not sure I've seen VLOG(0) before: what is the intended semantics? Same as VLOG(1)?

Reply:
This is used in a few files. So this isn't a first in TF. I want that to always be printed. VLOG(0) does that.